backends/events/sdl/sdl-events.cpp

```diff
@@ -912,22 +1108,21 @@ bool SdlEventSource::handleJoystickRemoved(const SDL_JoyDeviceEvent &device, Com
 
 int SdlEventSource::mapSDLControllerButtonToOSystem(Uint8 sdlButton) {
 	Common::JoystickButton osystemButtons[] = {
-	    Common::JOYSTICK_BUTTON_A,
-	    Common::JOYSTICK_BUTTON_B,
-	    Common::JOYSTICK_BUTTON_X,
-	    Common::JOYSTICK_BUTTON_Y,
-	    Common::JOYSTICK_BUTTON_BACK,
-	    Common::JOYSTICK_BUTTON_GUIDE,
-	    Common::JOYSTICK_BUTTON_START,
-	    Common::JOYSTICK_BUTTON_LEFT_STICK,
-	    Common::JOYSTICK_BUTTON_RIGHT_STICK,
-	    Common::JOYSTICK_BUTTON_LEFT_SHOULDER,
-	    Common::JOYSTICK_BUTTON_RIGHT_SHOULDER,
-	    Common::JOYSTICK_BUTTON_DPAD_UP,
-	    Common::JOYSTICK_BUTTON_DPAD_DOWN,
-	    Common::JOYSTICK_BUTTON_DPAD_LEFT,
-	    Common::JOYSTICK_BUTTON_DPAD_RIGHT
-	};
+		Common::JOYSTICK_BUTTON_A,
```

Comment:
These changes, if performed, have to be in a separate commit marked 'JANITORIAL: Indentation fixes', though, I'd advise avoiding them in this PR.